exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
+const greetingStart = "Hello, my name is";
+const name = "Petar";
 
-console.log(message);
+const greeting = greetingStart + " " + name;
+
```

Comment:
Your code it's great , it will be helpful   and readable if you delete  the  empty line . :)